project/chrome-extension/words.json

```diff
@@ -1,2 +1,2 @@
+{"COVID-19":{"description":"People of any age, even healthy young adults and children, can get COVID-19. Risk for severe illness with COVID-19 increases with age, with older adults at highest risk. Certain medical conditions can also increase risk for severe illness.","image":"/images/covid.png","URL":[{"source":"CDC: What You Need to Know","link":"https://www.cdc.gov/coronavirus/2019-ncov/your-health/need-to-know.html"},{"source":"CDC: Older Adults","link":"https://www.cdc.gov/coronavirus/2019-ncov/need-extra-precautions/older-adults.html"}]},"Elections":{"description":"There weren’t more votes than registered voters in the 2020 election.Experts say as many as 215 million voters were registered for the 2020 election.That far outnumbers the 159 million ballots cast.","image":"/images/elections.png","URL":[{"source":"USAToday","link":"https://www.usatoday.com/story/news/factcheck/2020/12/30/fact-check-fals-president-than-were-registered-u-s/4010087001/"},{"source":"Snopes","link":"https://www.snopes.com/fact-check/simple-math-on-biden-votes/"}]},"Climate Change":{"description":"People who study Earth see that Earth\u0027s climate is getting warmer. Earth\u0027s temperature has gone up about one degree Fahrenheit in the last 100 years.This may not seem like much. But small changes in Earth\u0027s temperature can have big effects.","image":"/images/climate-change.png","URL":[{"source":"NASA on Climate Change","link":"https://www.nasa.gov/audience/forstudents/k-4/stories/nasa-knows/what-is-climate-change-k4.html"},{"source":"NASA: Earth\u0027s energy budget","link":"https://climate.nasa.gov/news/3072/direct-observations-confirm-that-humans-are-throwing-earths-energy-budget-off-balance/"}]},"Vaccines":{"description":"There is currently no evidence of an increase in miscarriages as a result of COVID-19 immunization. This claim was made based on United Kingdom data.Health officials in the U.K. said the figures are being taken out of context. The article doesn’t account for the increase in the vaccination rates over time, or the average number of miscarriages that occur naturally.","image":"/images/vaccine.png","URL":[{"source":"CDC: Pregnancy Recommendations","link":"https://www.cdc.gov/coronavirus/2019-ncov/vaccines/recommendations/pregnancy.html"},{"source":"Reuters","link":"https://www.reuters.com/article/factcheck-miscarriage-vaccine/fact-check-there-is-no-evidence-to-suggest-covid-19-vaccines-increase-the-risk-of-miscarriage-idUSL1N2LT21A"}]}}
```

Comment:
The pull request in the json-update branch should have reflected these same changes